db/seeds/dev/data/palettes.csv

```diff
@@ -0,0 +1,19 @@
+pallet_name,project_id,color0,color1,color2,color3,color4

```

Comment:
Need to make sure we're being consistent with how we spell the word "palette". In the seed data, we have it as "pallet". Should be "palette". Other than that, looks great!